build/index.html

```diff
@@ -5,11 +5,20 @@
 
     <title>Let.Life</title>
 
+   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="vendors/angular-material/angular-material.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
 
+
+
+<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
```

Comment:
hai messo jquery in ./app/ e poi qui lo carichi da remoto.... secondo me la cosa migliore sarebbe prenderlo con bower già che ci siamo